src/en_us/index.js

```diff
@@ -166,28 +198,36 @@ export const i19editSeoFields = 'Edit SEO fields'
 export const i19editVariation = 'Edit variation'
 export const i19email = 'Email'
 export const i19emailAddress = 'Email address'
-export const i19emailCartText = 'You can finish it now, while the cart is available.'
-export const i19emailCartTitle = 'You did not finish your purchase!'
-export const i19emailDeliveredText = 'We hope you have enjoyed shopping with us and will soon visit our website again. The order was delivered to the address below'
-export const i19emailDeliveredTitle = 'Your package was delivered!'
+export const i19emailCartTextMsg = 'You can finish it now, while the cart is available.'
+export const i19emailCartTitleMsg = 'You did not finish your purchase!'
+export const i19emailDeliveredTextMsg = 'We hope you have enjoyed shopping with us and will soon visit our website again. The order was delivered to the address below'
+export const i19emailDeliveredTitleMsg = 'Your package was delivered!'
 export const i19emailInfo = 'Any update to the order will be notified by email, you can also track the progress through'
 export const i19emailMarketing = 'Email marketing'
 export const i19emails = 'Emails'
-export const i19emailShippedText = 'Your package is now with the carrier. To track the delivery progress, go to'
-export const i19emailShippedTitle = 'Your package is on the way!'
+export const i19emailShippedTextMsg = 'Your package is now with the carrier. To track the delivery progress, go to'
+export const i19emailShippedTitleMsg = 'Your package is on the way!'
 export const i19emptyCart = 'Your shopping cart is empty'
 export const i19enabled = 'Enabled'
+export const i19end = 'End'
+export const i19endDate = 'End date'
 export const i19endpoint = 'Endpoint'
 export const i19enterYourDocNumber = 'Enter your document number to confirm'
 export const i19enterYourEmail = 'Enter your email to proceed'
 export const i19erp = 'ERP'
+export const i19error = 'Error'
+export const i19errorMsg = 'Unfortunately an error has occurred in your application'
```

Comment:
Pelo nome da constante deveria ser uma mensagem genérica aqui, tipo:

```suggestion
export const i19errorMsg = 'An error has occurred, try again or contact us'
```